TESTING.md

```diff
@@ -169,3 +169,57 @@ We have a suite of external snaps tests that runs each night using the latest sn
 This is an experimental suite, with still some details to define. The idea is to build a snap recording a manifest of all the details of the build. Then build the snap again, but this time using the manifest instead of the source `snapcraft.yaml`, and compare that both snaps are equal.
 
 Currently, the suite is using the snaps of the integration suite to check the reproducibility. It is located in https://github.com/elopio/snapcraft-reproducible/
+
+## Testing arm
+
+It is possible to emulate an arm64 machine on an amd64 host, which is very useful for running manual exploratory tests for snapcraft. To set it up:
+
+1. Download the latest ubuntu arm64 uefi image from https://cloud-images.ubuntu.com/releases/16.04/release/
+2. Keep a pristine copy of the image, in case you want to reset the machine, replacing <UBUNTU_IMAGE> with the name of the file you downloaded on step 1:
+
+    $ cp <UBUNTU_IMAGE> <UBUNTU_IMAGE>.pristine
+
+3. Download the latest UEFI firmware image QEMU_EFI.fd from https://releases.linaro.org/components/kernel/uefi-linaro/latest/release/qemu64/
+4. Create a cloud init file, replacing <USER> and <SSH_PUBLIC_KEY> with your values:
+
+    $ cat > cloud.txt << EOF
+    #cloud-config
+    users:
+      - name: <USER>
+        ssh-authorized-keys:
+          - ssh-rsa <SSH_PUBLIC_KEY>
+        sudo: ['ALL=(ALL) NOPASSWD:ALL']
+        groups: sudo
+        shell: /bin/bash
+    EOF
+
+5. Create a cloud-config disk image:
+
+    $ sudo apt install --yes cloud-image-utils
+    $ cloud-localds --disk-format qcow2 cloud.img cloud.txt
```

Comment:
is `cloud-img` really not `<UBUNTU_IMAGE>`, also, this should be called here and there `<ubuntu-image>`

Reply:
This is a different disk, take a look at the command call that you use both cloud image and ubuntu image. I will change the format of the params.